src/Core/src/Handlers/View/ViewHandlerOfT.Tizen.cs

```diff
@@ -27,7 +27,7 @@ public abstract partial class ViewHandler<TVirtualView, TPlatformView> : IPlatfo
 
 		public IPlatformViewHandler? Parent { get; private set; }
 
-		public EvasObject NativeParent => MauiContext?.GetNativeParent() ?? throw new InvalidOperationException($"NativeParent cannot be null here");
+		public EvasObject PlatformParent => MauiContext?.GetPlatformParent() ?? throw new InvalidOperationException($"PlatformParent cannot be null here");
```

Comment:
This appears to be the only breaking change but it should be ok because I am not sure we have that many Maui Tizen libs just yet since we just released RC with this.

Reply:
There are a couple of other similar properties, but they are protected and on Shell things. Even less likely to be hit.